src/HotChocolate/Core/src/Types.Scalars/ThrowHelper.cs

```diff
@@ -340,5 +340,27 @@ public static SerializationException IPv4Type_ParseLiteral_IsInvalid(IType type)
                     .Build(),
                 type);
         }
+
+        public static SerializationException RegularExpressionType_ParseValue_IsInvalid(IType type)
+        {
+            return new SerializationException(
+                ErrorBuilder.New()
+                    .SetMessage(ScalarResources.RegularExpressionType_IsInvalid_ParseValue)
+                    .SetCode(ErrorCodes.Scalars.InvalidRuntimeType)
+                    .SetExtension("actualType", WellKnownScalarTypes.RegularExpression)
```

Comment:
`.SetExtension("pattern", validationPattern)`

though only do this if the regex has a sane length. The IPv4 pattern or the Email pattern are to big :D

Reply:
what would be then a sane lenght? should we...
1. Cut to a sane pattern all the patterns for this SetExtension method... if sane is 100 characters, then 98 plus ".."
2. Add the full pattern into the error message?
would this do?